src/operations.rs

```diff
@@ -51,18 +51,15 @@ pub(crate) fn crypto_kem_enc<R: CryptoRng + RngCore>(
     pk: &[u8; CRYPTO_PUBLICKEYBYTES],
     rng: &mut R,
 ) {
-    let mut two_e = [0u8; 1 + SYS_N / 8];
-    two_e[0] = 2;
```

Comment:
since you change the size and the content of the array `two_e` this adaption needs to be done aswell in the `test_encrypt` function in `encrypt.rs`. Moreover the compared ciphertext bytes `mceliece8192128f_encrypt_ct` need to be newly generated and modified in the `testdata.txt` file